src/plugins/plot/src/telemetry/PlotController.js

```diff
@@ -71,6 +71,7 @@ define([
         this.config.series.forEach(this.addSeries, this);
 
         this.followTimeConductor();
+        window.plot = this;
```

Comment:
@larkin This line should be removed, otherwise changes look good.

There are no tests written for the new controllers. Is there a plan to add tests at a later time?

Reply:
There is an issue for backfill.